src/components/banner.svelte

```diff
@@ -231,28 +84,27 @@
     }
   })
 
-  function setCookie (choices) {
+  function setCookie(choices) {
```

Comment:
I can't remember if I linted this or not, but the @beyonk/eslint-config should (I think) pick up that there needs to be a space between function name and brackets.

Not to worry though, I can just lint auto-fix it before merge.

Reply:
Sorry, this seems to be my prettier config. I didn't bother to actually set it up with your eslint config so it did what it did. :) We could lint it properly if you want to.

Reply:
Yeah don't worry about it too much - I don't lint all my projects so there is a chance that it doesn't even have linting installed - I will rectify the linting post merge :)